turbopack/crates/turbopack-core/src/chunk/mod.rs

```diff
@@ -384,7 +392,7 @@ async fn graph_node_to_referenced_nodes(
                 .await?
                 .into_iter()
                 .map(|&module| async move {
-                    if chunking_type == ChunkingType::Traced {
+                    if matches!(chunking_type, ChunkingType::Traced) {
```

Comment:
I feel like `ChunkingType::Traced` shouldn't exist at all and traced references shouldn't implement ChunkableModuleReference but TracedModuleReference instead, but that's something for another PR